cepheus-cep/src/main/java/com/orange/cepheus/cep/model/Provider.java

```diff
@@ -44,15 +54,18 @@
     Provider(String url) {
         this.url = url;
     }
+    public Provider(){
```

Comment:
Leave an empty line before each method (code convention of the rest of the project)